src/test/java/picard/analysis/TheoreticalSensitivityTest.java

```diff
@@ -268,7 +268,7 @@ public void testSensitivityAtConstantDepth(final double expected, final File met
         // keeping the compute time of the tests short.
         return new Object[][]{
                 {0.90, wgsMetricsFile, 0.5, 400},
-                {0.77, wgsMetricsFile, 0.3, 400},
+                {0.78, wgsMetricsFile, 0.3, 400},
```

Comment:
why did this change?

Reply:
The reason is because it was on the edge between .77 and .78.  The bug-fix causes the value to go up because it no longer trims the highest base quality score from the distribution.

It's an integration test so I didn't calculate this value independently.  In a way, that makes me wonder about just how valuable this particular test is, as well as the test that uses theoreticalSensitivityDataProvider in CollectTargetedMetricsTest.java.